packages/lib/src/index.ts

```diff
@@ -5,10 +5,11 @@ if (process.env.NODE_ENV === 'development') {
     // require('preact/debug');
 }
 
+import {CoreOptions} from 'src/core/types';
```

Comment:
Likely prettier is not setup on your env :)

```suggestion
import { CoreOptions } from 'src/core/types';
```

Reply:
You are right, I thought it was configured as our prettier config does remove the spaces between the braces. Anyhow, fixed it.